vipps-report-api-faq.md

```diff
@@ -95,6 +94,22 @@ We may add display of the Ledger ID on
 [portal.vipps.no](https://portal.vipps.no)
 in the future.
 
+## How can I get the details for a payment I see in my bank?
+
+The payout details can be found with the `/funds` endpoint, either under date or
+using the feed. Match the settlement id in the bank with `pspReference`.
+
+The Report API is designed to provide updated data independently of the bank payments. There are several reasons for this, including:
+
+1. A merchant can have a negative balance and not get settlement payouts for a long time.
+   Periodic checks of settlements would thus not make sense until the merchant has a positive again.
+1. Merchants have different risk profiles (or could have at least). If we deemed that we would hold
+   back some of the balance from payouts to an airline company.
+   Let's say they sell three tickets at 5000 NOK each. We then change their risk profile saying we
+   need to hold back 7500 NOK and then the settlement run. We will then pay out 7500 NOK
+   (if there are no fees). Now: What captures will then be part of that settlement?
+1. We may, in rare cases, manually correct payouts. 
```

Comment:
Maybe describe more. I don't know how such corrections are made but if we do mention them then perhaps explain how such corrections happen to avoid uncertainty among merchants that we might suddenly do corrections without informing them properly.

Reply:
As long as the change is an improvement, I think it's fine to merge. We can improve more any time.